vggish.py

```diff
@@ -19,12 +19,11 @@
 from keras.engine.topology import get_source_inputs
 from keras import backend as K
 
-from sound import vggish_params2 as params
+import vggish_params as params
 
 
 # weight path
-WEIGHTS_PATH = '/mount/hudi/moe/moce/soundnet_moce/sound/weights/vggish_audioset_weights_without_fc2.h5'
-WEIGHTS_PATH_TOP = '/mount/hudi/moe/moce/soundnet_moce/sound/weights/vggish_audioset_weights.h5'
+WEIGHTS_PATH = '/home/brain/Documents/git/VGGish/vggish_audioset_weights_without_fc2.h5'
```

Comment:
Consider that on lines 110-115 of this module, `WEIGHTS_PATH_TOP` is being used so you should keep that variable defined. It looks like just a small mistake